src/Components/Actions/ActionsButton.css

```diff
@@ -0,0 +1,7 @@
+#actionsButton {
+  background-color: khaki;
```

Comment:
maybe it's a good time to use SCSS, Mostly for the color consts.
I think that the sooner you will implement it the easier it will be in the future.

Reply:
I guess it wont hurt, but what is wrong with this line?